jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpStatus.java

```diff
@@ -381,11 +381,27 @@ public static boolean isSuccess(int code)
      * @return true if within range of codes that belongs to
      * <code>Redirection</code> messages.
      */
-    public static boolean isRedirection(int code)
+    public static boolean isRedirect(int code)
```

Comment:
The RFC naming is actually "redirection", as also the javadoc of this method uses.

I would keep this method named `isRedirection()` with 300-399, and just add `isLocationRedirect()` or `isRedirectWithLocation()` (I prefer the former).